smoke-tests/springboot/build.gradle

```
@@ -0,0 +1,23 @@
+plugins {
+  id 'org.springframework.boot' version '2.3.2.RELEASE'
+  id 'io.spring.dependency-management' version '1.0.9.RELEASE'
+  id 'java'
+  id 'com.google.cloud.tools.jib' version '2.4.0'
+}
+
+group = 'io.opentelemetry'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '11'
```

i think nice to keep our smoke tests targeted to Java 8, even if we are only running them on Java 11 currently
```suggestion
sourceCompatibility = '8'
```